cloud/keystone.go

```diff
@@ -31,6 +32,10 @@ import (
 	"github.com/gophercloud/gophercloud/pagination"
 )
 
+var (
+	log = l.NewLogger()
```

Comment:
It's a bit terrifying that right now we have **0** log messages from this file.

Reply:
I added one more log message about `Invalid token`, because we have seen such errors lately. Other errors are improbable or quite obvious so I will leave them for now.